composer.json

```diff
@@ -28,6 +32,13 @@
         }
     },
     "require-dev": {
-        "sllh/php-cs-fixer-styleci-bridge": "^2.1"
+        "matthiasnoback/symfony-config-test": "^3.0",
+        "php-http/guzzle6-adapter": "^1.1",
+        "sllh/php-cs-fixer-styleci-bridge": "^2.1",
+        "symfony/dependency-injection": "^2.8 || ^3.0",
+        "symfony/http-kernel": "^2.8 || ^3.0"
+    },
+    "suggest": {
+        "php-http/httplug-bundle": "For Symfony integration as a bundle"
```

Comment:
"as a bundle with lstrojny/fxmlrpc".

BTW, we should not suggest it now, but this is just for info, so okay.